src/components/sendReadable/send.js

```diff
@@ -119,6 +119,15 @@ class SendReadable extends React.Component {
               onChange={this.handleChange.bind(this, 'recipient')}
               disabled={true}
             />
+            {this.state.reference.value ?
```

Comment:
Can't we just use some p> with styles?
also, name `reference` for the Junit tests is a bit generic